samples/server/petstore/nodejs/api/swagger.yaml

```diff
@@ -356,8 +356,8 @@ paths:
         description: "ID of pet that needs to be fetched"
         required: true
         type: "integer"
-        maximum: 5
-        minimum: 1
+        maximum: 5.0
+        minimum: 1.0
```

Comment:
@fehguy The output is now a decimal instead of integer: https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/test/resources/2_0/petstore.yaml#L343-L344

I don't think it's related to the change in this PR, probably due to changes in the Swagger Parser.